eng/common/scripts/stress-testing/stress-test-deployment-lib.ps1

```diff
@@ -159,8 +159,9 @@ function DeployStressPackage(
     $dockerFilePath = "$($pkg.Directory)/Dockerfile"
     if ($pushImages -and (Test-Path $dockerFilePath)) {
         Write-Host "Building and pushing stress test docker image '$imageTag'"
-        $dockerFile = Get-ChildItem $dockerFilePath
-        Run docker build -t $imageTag -f $dockerFile.FullName $dockerFile.DirectoryName
+        $dockerFile = Get-ChildItem "$($pkg.Directory)/Dockerfile"
```

Comment:
This will need to be removed. @ckairen do you think you would be able to get to https://github.com/Azure/azure-sdk-tools/issues/2569 this week?

Reply:
I think it's just a matter of copying these changes and adding some docs for the new chart annotation fields located [here](https://github.com/Azure/azure-sdk-for-go/pull/16831/files#diff-fa372fad5b3dbf6bab1bb08336692324876f4c04553e99e28c4ba381dd1e48dbR9-R10).

Reply:
Yeah, it's in here so what's in the branch can deploy, but it's not meant to be permanent.

Reply:
Yeah only reason I bring it up is our tooling will prevent you from merging due to changes in eng/common.

Reply:
Ok, I'll rush this this week

Reply:
https://github.com/Azure/azure-sdk-tools/pull/2599 Here's the PR :)